package.json

```diff
@@ -50,7 +50,7 @@
     "s3": "^4.4.0",
     "scrypt": "^6.0.3",
     "tabookey-gasless": "^0.3.3-a",
-    "web3": "1.0.0-beta.33"
+    "web3": "1.0.0-beta.55"
```

Comment:
Ah, sorry I had approved already.

Bit hesitant now to just bump the web3 version as those beta versions all have their own weird little bugs and quirks. Why did you do that?
@austintgriffith @dmihal thoughts?

Reply:
Hey @TimDaub! Thanks for the review. One of the reasons I went with it is because `burner-core` seems to be using it: https://github.com/austintgriffith/burner-core/blob/develop/core/package.json#L10

If you think it's not safe,  I can back out that commit.